.github/workflows/release.yml

```diff
@@ -0,0 +1,34 @@
+---
+name: "Release"
+
+on:
+  push:
+    tags:
+      - 'v[0-9]+.[0-9]+.[0-9]+'
+
+jobs:
+  distribute:
+    name: Distribute
+    runs-on: ubuntu-latest
+    steps:
+      - name: 'Check out repository'
+        uses: actions/checkout@v2
+      - name: 'Publish the release to GitHub'
+        uses: actions/create-release@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          tag_name: ${{ github.ref }}
+          release_name: Tsunami ${{ github.ref }}
```

Comment:
This is slightly different from the `0.0.3` release, as this would have the `v` preface in the release name

Reply:
This is fine. We will keep the convention of adding `v` to the release name from now on.